lib/src/androidTest/kotlin/at/bitfire/ical4android/impl/TestTask.kt

```diff
@@ -12,14 +12,15 @@ import at.bitfire.ical4android.DmfsTask
 import at.bitfire.ical4android.DmfsTaskFactory
 import at.bitfire.ical4android.DmfsTaskList
 import at.bitfire.ical4android.Task
+import java.util.UUID
 
 class TestTask: DmfsTask {
 
     constructor(taskList: DmfsTaskList<DmfsTask>, values: ContentValues)
         : super(taskList, values)
 
     constructor(taskList: TestTaskList, task: Task)
-        : super(taskList, task)
+        : super(taskList, task, UUID.randomUUID().toString(), null, null, 0)
```

Comment:
Using random UUIDs for tests has always been a headache for me, to be honest. Even though UUIDv4 is considered unique, I think it's a good practise to just hardcode a random UUID, for reproducibility.